Examples/Scripts/PreFreeSurferPipelineBatch.sh

```diff
@@ -252,9 +252,18 @@ main()
 		#     Average any repeats and use Spin Echo Field Maps for readout
 		#     distortion correction
 		#
-		#   "GeneralElectricFieldMap"
-		#     Average any repeats and use General Electric specific Gradient
-		#     Echo Field Map for readout distortion correction
+		#   "GEHealthCareLegacyFieldMap"
+		#     Average any repeats and use GE HeathCare Legacy specific Gradient
+		#     Echo Field Map for readout distortion correction. 
+		#	  The Legacy fieldmap is a two volume file: 1. field map in Hz and 2. magnitude.
+		# 	  Use "GEB0InputName" variable to specify the 2-Volume file. 
+		#	  Set "TE" variable to the EchoTime difference (TE2-TE1). 
```

Comment:
This could be confusing.  If "TE" is really supposed to "DeltaTE", why not rename the variable to "DeltaTE" instead.

Reply:
I'll fix that